### PR TITLE
images: Move to current Debian stable version 12 "bookworm"

### PR DESCRIPTION
**Pull request by martinpitt**
* [x] image-refresh debian-stable
 * [x] image-refresh debian-testing
 * Land in lockstep with https://github.com/cockpit-project/cockpit/pull/18975 (cockpit tests run there) and https://github.com/cockpit-project/cockpit-machines/pull/1109 (machines tests run there)

### CODE REVIEW

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230617-050606.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230617-050808.log

**Comment by jelly** [comment]
For debian-testing:

```
The following packages have unmet dependencies:
 freeipa-client : Conflicts: systemd-timesyncd but 252.6-1 is to be installed
                  Recommends: chrony but it is not installable
```

Debian-stable

```
Get:4 https://deb.debian.org/debian bookworm-backports InRelease [49.7 kB]
Err:11 http://debug.mirrors.debian.org/debian-debug -debug Release
  404  Not Found [IP: 130.242.6.199 80]
```

Retrying...

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230617-084343.log

**Comment by jelly** [comment]
Hmm 404 seems persistent:

```
Err:8 http://debug.mirrors.debian.org/debian-debug -debug Release
  404  Not Found [IP: 217.196.149.232 80]
```

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230617-095151.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230617-095757.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230617-143535.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230617-152323.log

**Comment by martinpitt** [comment]
I ran the image build manually, and indeed there is a group `netdev:x:1002:debian` which is completely wrong. I checked the postinst scripts, and the only packages that create the group are wpasupplicant and network-manager, and they both use `--system` correctly.

I reported this to the debian-cloud ML and added a workaround.

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230620-061717.log

**Comment by cockpituous** [comment]
image-refresh debian-stable done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-stable-20230620-065838

**Comment by cockpituous** [comment]
image-refresh debian-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-testing-20230620-072531

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230620-072525.log

File: images/scripts/debian.setup [hunk 1]
```diff
@@ -105,7 +104,11 @@ systemd-timesyncd
 # wait until cloud-init finishes, so that it doesn't clobber sources.list
 until systemctl list-jobs | grep -q "No jobs"; do sleep 1; done
 
-RELEASE=$(grep -m1 ^deb /etc/apt/sources.list | awk '{print $3}')
+# HACK: bookworm cloud images have broken netdev group
+# https://lists.debian.org/debian-cloud/2023/06/msg00029.html
```
Comment (martinpitt) [inline images/scripts/debian.setup:108]: In case I push again, link to https://bugs.debian.org/1038691

Comment (martinpitt) [reply images/scripts/debian.setup]: Done.

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230620-090505.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230620-090707.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230620-091111.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230620-091313.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230620-091313.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230620-092929.log

**Comment by cockpituous** [comment]
image-refresh debian-stable done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-stable-20230620-101704

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230620-101717.log

**Comment by cockpituous** [comment]
image-refresh debian-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-testing-20230620-105053

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230620-105050.log

**Pull request merged**